xenmgr/XenMgr/Config.hs

```diff
@@ -269,7 +270,9 @@ appSetEnableDom0Networking True =
       exist <- doesFileExist "/config/system/dom0-networking-disabled"
       if exist then removeFile "/config/system/dom0-networking-disabled" else return ()
 appSetEnableDom0Networking False =
-    liftIO $ writeFile "/config/system/dom0-networking-disabled" ""
+    liftIO $ do
+      createDirectoryIfMissing True "/config/system"
```

Comment:
This must have relied on ```/config/system``` being created somewhere else.  Tough to say where exactly but this is 100% fine to add as a defensive check.